syft/frameworks/crypten/context.py

```diff
@@ -87,7 +87,7 @@ def _send_party_info(worker, rank, msg, return_values, model=None):
     return_values[rank] = utils.unpack_values(response.object, model)
 
 
-def run_multiworkers(workers: list, master_addr: str, master_port: int = 15463):
```

Comment:
Q: I didn't get why you always change ports George :p

Reply:
Ahh...Because while testing I get the error "port already in use" (if the tests fail...I can change it back)

Reply:
Done

Reply:
Ah, that's because you have processes still running in the background. I have a script in my workspace that I call if processes keeps running, it's a single command script, might be helpful:

```bash
#!/bin/bash

kill -9 `ps |grep python | awk '{$1=$1};1' |cut -f1 -d ' '`
```